web-common/src/main/java/edu/unc/lib/boxc/web/common/utils/AnalyticsTrackerUtil.java

```diff
@@ -217,9 +222,6 @@ public void run() {
             if (label != null) {
                 params.add(new BasicNameValuePair("el", label));
             }
-            if (value != null) {
```

Comment:
why was this no longer needed?

Reply:
It was never used, so I figured I might as well remove the dead code